### DISCUSSION TITLE
Install New Software DBeaver version upgrade fails - Transaction Locks Monitor conflict

### DISCUSSION THREAD

**Discussion by dstidham63** [dbeaver/dbeaver#21081] [DBeaver Development] [answered]
Today, I tried to upgrade 23.1.1 to 23.1.5 using the Install New Software feature and was unsuccessful due a dependency conflict.  Please see attached screenshot.

In short, it reports that only one can be installed for the DBeaver Transaction Locks Monitor UI and shows two different versions.  Is there a work around?

![thumbnail_image001](https://github.com/dbeaver/dbeaver/assets/73491425/104f8832-4d38-44c6-a081-822d214bfa9c)

**Comment by E1izabeth** [comment]
Are you using standalone application or eclipse plugin?

**Reply by dstidham63** [reply]
Standalone DBeaver client.

**Reply by E1izabeth** [reply]
How exactly are you performing update? Are you doing it this way?
![image](https://github.com/dbeaver/dbeaver/assets/28875055/f9dc9916-002c-4369-8010-6813682eb320)

**Reply by dstidham63** [reply]
No, I am not using that repository.  It is not listed as part of the default sites in DBeaver CE 23.1.1.

![thumbnail_image001 (2)](https://github.com/dbeaver/dbeaver/assets/73491425/fe408947-bead-454e-b39f-9d74da72648f)

This is how I am doing it -
![thumbnail_image001 (1)](https://github.com/dbeaver/dbeaver/assets/73491425/e872bbee-7546-495c-8d25-fc8d708af323)

**Comment by E1izabeth** [comment, marked as answer]
I reproduced the issue. Thanks.
You can use Check for updates button for update
![image](https://github.com/dbeaver/dbeaver/assets/28875055/5c196e28-c335-414b-9ef5-ba7763da99fb)

**Reply by dstidham63** [reply]
Within my work environment, the Check for Updates feature fails.

![image](https://github.com/dbeaver/dbeaver/assets/73491425/7288d11d-0934-45f1-b3bc-10a221db9b8e)

**Reply by dstidham63** [reply]
Any suggestions for a work around?

**Reply by E1izabeth** [reply]
Can you open this website (https://dbeaver.io/) with your web browser?

**Reply by dstidham63** [reply]
I supposed I should have included that while I can access the website for DBeaver CE, downloading and installed doesn't work within the work landscape, much like the Check for Updates feature within the application fails.

I had hoped that I could get the application to update itself rather than install a new version (i.e. setup executable) - which a lack of administrator permissions is preventing.

**Comment by E1izabeth** [comment]
I'm not sure DBeaver is supposed to be updated like this. Has it been ever working for you this way?

**Reply by dstidham63** [reply]
This is the first time I have attempted to update this way.
